app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    result = {}
+
+    def wrapper(*args, **kwargs) -> None:
```

Comment:
Indicate in the return type annotation that `wrapper` could return `Any` to match the wrapped function output